app/routes/tags.rb

```diff
@@ -0,0 +1,13 @@
+module ExercismWeb
+  module Routes
+    class Tags < Core
+      get "/tags/search" do
+        please_login
+
+        tags = Tag.find_by_similarity(params["q"])
+
+        json tags
```

Comment:
Should we delete this file, move this to the API and make the javascript call the API directly? How to enforce a logged user in this case?